numpy/polynomial/_polybase.py

```diff
@@ -919,10 +919,8 @@ def fit(cls, x, y, deg, domain=None, rcond=None, full=False, w=None,
         ----------
         x : array_like, shape (M,)
             x-coordinates of the M sample points ``(x[i], y[i])``.
-        y : array_like, shape (M,) or (M, K)
-            y-coordinates of the sample points. Several data sets of sample
-            points sharing the same x-coordinates can be fitted at once by
-            passing in a 2D-array that contains one dataset per column.
+        y : array_like, shape (M,)
+            y-coordinates of the sample points.
```

Comment:
```suggestion
            y-coordinates of the sample points ``(x[i], y[i])``.
```
Might as well add this extra wording to show consistency between the parameters, now that all of the extra wording is removed.